src/pip/_internal/index/package_finder.py

```diff
@@ -696,11 +696,9 @@ def set_prefer_binary(self):
 
     def skipped_links_requires_python(self):
         # type: () -> List[str]
-        skips = [skip[1] for skip in self._logged_links if 'Requires-Python' in skip[1]]
-        # cleans duplicate package version found from different links
-        skips = list(dict.fromkeys(skips))
-        skips.sort()
-        return skips
+        return sorted(
```

Comment:
updated, but lint complained so I had to break the line anyway. is this idiomatic?

Reply:
Yeah I think this is good